lti_consumer/lti_consumer.py

```diff
@@ -270,6 +274,43 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
         default="",
         scope=Scope.settings
     )
+
+    # LTI 1.3 modified fields
+    lti_version = String(
+        display_name=_("LTI Version"),
+        scope=Scope.settings,
+        values=[
+            {"display_name": "LTI 1.1/1.2", "value": "lti_1p1"},
+            {"display_name": "LTI 1.3", "value": "lti_1p3"},
+        ],
+        default="lti_1p1",
+    )
+    lti_1p3_launch_url = String(
+        display_name=_("LTI 1.3 Launch URL"),
+        default='',
+        scope=Scope.settings
+    )
+    lti_1p3_oidc_url = String(
+        display_name=_("LTI 1.3 OIDC URL"),
+        default='',
+        scope=Scope.settings
+    )
+    lti_1p3_tool_public_key = String(
+        display_name=_("LTI 1.3 Tool Public Key"),
+        default='',
+        scope=Scope.settings
+    )
+    lti_1p3_platform_key = String(
+        display_name=_("LTI 1.3 Platform Key"),
+        default=RSA.generate(2048).export_key('PEM'),
+        scope=Scope.settings
+    )
+    lti_1p3_deployment_id = String(
```

Comment:
Deployment ID goes in the same place as the platform key.

Reply:
As we discussed earlier, the course settings in Studio might be an alternative, but UX will hurt.

Reply:
I've changed the implementation a little bit and the deployment id is now fixed.
Each XBlock has it's own Client Key and ID now.